app/lib/textExtractor.ts

```diff
@@ -0,0 +1,9 @@
+import officeparser from 'officeparser'
+
+export const extractTextFromPDF = async (pdfBuffer: Buffer) => {
```

Comment:
**issue**: Consider adding error handling for unsupported file types in `extractTextFromPDF`. Currently, it assumes the input is always a PDF, which might not be the case if the function is reused elsewhere.